mwclient/client.py

```diff
@@ -497,8 +494,8 @@ def raw_call(self, script, data, files=None, retry_on_error=True, http_method='P
                 stream = self.connection.request(http_method, url, **args)
                 if stream.headers.get('x-database-lag'):
                     wait_time = int(stream.headers.get('retry-after'))
-                    log.warning('Database lag exceeds max lag. '
-                                'Waiting for {} seconds'.format(wait_time))
+                    log.warning(f'Database lag exceeds max lag. '
+                                f'Waiting for {wait_time} seconds')
```

Comment:
istr that it's technically wrong to use `.format` *or* f-strings when using `logging`; you should always do it like `log.warning('lala %s', wait_time)` (which doesn't actually 'resolve' it at that time; it's done later elsewhere). there's some background on this at https://stackoverflow.com/questions/75715376/set-the-formatting-style-for-logging-percent-format-style-bracket-or-f-stri for e.g. so perhaps for all the cases of `log.*` we should make this pr use percent instead?

Reply:
see also https://docs.python.org/3/howto/logging-cookbook.html#using-particular-formatting-styles-throughout-your-application

Reply:
Yep good catch, I've updated the two relevant locations.